### ISSUE TITLE
add a Pipfile

### ISSUE THREAD

**Issue by lynxis** [lynxis/keyblepy#1]
there is no pipfile with all dependencies.

**Comment by yunnanpl** [comment]
Yes, that would be useful, at the moment I needed to install those:

- bluepy
- transitions
- pycrypto (or theoretically better pycryptodome)
- transitions
